index.htm

```diff
@@ -368,11 +289,11 @@ <h4>Alternative Linux sources</h4>
             </div>
             <!--/GRC Research Downloads-->
             <!--MISC-->
-            <div class="col-xs-12 col-sm-6">
+            <div class="col-12 col-sm-6">
                 <h4>Other downloads</h4>
-                    <a class="btn btn-round grcbtn" href="https://boinc.berkeley.edu/download_all.php">BOINC Clients</a><br/>
-                    <a class="btn btn-round grcbtn" href="https://boinc.berkeley.edu/wiki/Add-on_software">BOINC Add-on software</a><br/>
-                    <a class="btn btn-round grcbtn" href="https://github.com/marius311/boinc-server-docker">BOINC server Docker container</a>
+                    <a class="btn rounded-pill grcbtn" href="https://boinc.berkeley.edu/download_all.php">BOINC Clients</a><br/>

```

Comment:
The other merged `btn-round` download links came back with the NOW-UI class. Could use `rounded-pill` as well so the font weight matches--that class will go away anyway.